src/textual/screen.py

```diff
@@ -91,7 +115,7 @@ def __init__(
         self,
         requester: MessagePump,
         callback: ScreenResultCallbackType[ScreenResultType] | None,
-        future: asyncio.Future[ScreenResultType] | None = None,
+        future: AwaitScreen[ScreenResultType] | None = None,
```

Comment:
Same here. Since it is no longer a future, it deserves a better name.

My bad. `future` was a terrible name, since it presumed the implementation.

Reply:
do you have an idea for the name? I was going to go for `await_screen` ?